File: OCH_Tao/ABC/B/351.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(input.shift(), 10);
let A = [];
let B = [];
for (let i = 0; i < N; i++) {
  A.push([...input.shift()]);
}
for (let i = 0; i < N; i++) {
  B.push([...input.shift()]);
}
for (let i = 0; i < N; i++) {
  for (let j = 0; j < N; j++) {
    if (A[i][j] !== B[i][j]) {
      console.log(`${i + 1} ${j + 1}`);
      break;
    }
  }
}

File: OCH_Tao/ABC/B/351.py
N = int(input())
A = []
for i in range(N):
  A.append(list(input()))
B = []
for i in range(N):
  B.append(list(input()))
for i in range(N):
  for j in range(N):
    if A[i][j]!=B[i][j]:
      print(f"{i+1} {j+1}")
      break